packages/documentation/docs/controls/_select_styleguide.md

```diff
@@ -0,0 +1,66 @@
+Selects provide users to make a single or multi selection from a filterable list. With the editable variant, users can also add new options to the list.
+
+![Overview](https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?node-id=3054-593&t=6gU2IFkPWR943af8-4)
+
+1. Label
+2. Required indicator
+3. Placeholder
+4. Clear button
+5. Open dropdown button
+6. Container
+7. Dropdown list
+8. List header
+9. Selected list item
+10. Add new option (editable mode)
+
+## Options
+
+- **Label**: See [form field](forms-field.md).
+- **Placeholder**: Use a placeholder to provide a hint about what to enter or additional relevant context while the input field is empty. We typically use a placeholder when the label is not visible or we need to provide additional context.
+- **Helper text**: See [form field](forms-field.md).
+- **Feedback text**: See [form field](forms-field.md).
+- **Show clear button**: Selects can have a dedicated button to easily clear the selection. Hide the button when you offer other ways to reset (e.g. a default option like "none"), or if you aim for a simplified keyboard accessibility.
+- **Select list header:** Use a header to provide additional context or instructions about the options to help users understand the choices better. 
+- **Hide list header**: Hide the header of the dropdown list when it is not needed.
+- **Hint for no matches**: Set a message that is displayed when no item matches the inserted text.
+- **Editable**: Allow users to add new options to the list.
+- **Multi-select**: Allow users to select multiple items from the list.
+## Behavior in context
+
+- **Validation**: See [validation](forms-validation.md).
+- **Interaction**:
+	- Click or Enter key on button opens dropdown list.
+	- Typing in the input field filters the dropdown list.
+	- Arrow keys navigate within dropdown list.
+	- Click or enter selects highlighted list item.
+	- Escape key closes dropdown list and returns to the originally selected value.
+- **Overflow**: 
+	- The text in an input field is truncated with the length of the container.
+	- On the multi-select, the selected items break into a second line and then shows a scrollbar if it grows beyond.
+	- The dropdown list is scrollable when the list exceeds the container height. Its width is defined by the longest item.
+- **Alignment**: Selects are always aligned to the left, while right alignment is reserved exclusively for [number fields](number-input,md).
+## States
+
+The select field has five states: default, hover, active/focused, disabled and read-only. In the disabled state, the input field is displayed without offering any user interaction.
+
+![Field States](https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?node-id=3198-7167&t=EBVCuGpWXmdVYgeZ-4)
```

Comment:
[Select States] should link to https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?node-id=3960-760&t=DtCmoFcLwhf7ke3S-4